packages/standard-components/src/forms/RelationshipField.svelte

```diff
@@ -16,8 +17,26 @@
   // Picker state
   let options = []
   let tableDefinition
+  let fieldText = ""
+  
+  const setFieldText = (value) => {
+    if (fieldSchema?.relationshipType === 'one-to-many') {
+      if (value?.length && options?.length) {
+        const row = options.find(row => row._id === value[0])
+        return row.name
+      } else {
+        return placeholder  || 'Choose an option'
+      } 
+    } else {
+      if (value?.length) {
+        return `${value?.length ?? 0} selected rows`
+      } else {
+        return placeholder  || 'Choose some options'
+      } 
+    }
+  }
 
-  $: fieldText = `${$fieldState?.value?.length ?? 0} selected rows`
+  $: options, fieldText = setFieldText($fieldState?.value)
```

Comment:
I've never seen this syntax before, what does this do by putting `options, ` here? Does it trigger a reaction whenever `options` changes?

Reply:
Yep! That's correct! 👍